miniwdl_aws.cfg

```diff
@@ -24,6 +24,12 @@ download_concurrency = 5
 # configure AWS Batch jobs to mount the filesystem at this same location.
 root = /mnt/efs
 
+[batch_client]
+default_retries = {
```

Comment:
Please put this in the existing `[aws]` section, call it something like `boto3_retries` to disambiguate it from WDL-level task retries, and add a descriptive comment referring to the [relevant boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html#defining-a-retry-configuration-in-a-config-object-for-your-boto3-client).